main/main.cpp

```diff
@@ -94,9 +94,12 @@
 #include "editor/editor_paths.h"
 #include "editor/editor_settings.h"
 #include "editor/editor_translation.h"
+#include "editor/plugins/packed_scene_translation_parser_plugin.h"
+#include "editor/pot_generator.h"
 #include "editor/progress_dialog.h"
 #include "editor/project_manager.h"
 #include "editor/register_editor_types.h"
+#include "modules/gdscript/editor/gdscript_translation_parser_plugin.h"
```

Comment:
```suggestion
#ifdef MODULE_GDSCRIPT_ENABLED
#include "modules/gdscript/editor/gdscript_translation_parser_plugin.h"
#endif // MODULE_GDSCRIPT_ENABLED
```
Need to ensure these are available